packages/microservice/src/bus-adapters/kafka-bus.adapter.ts

```diff
@@ -1,34 +1,36 @@
-import {
-  Consumer,
-  Producer,
-} from "@nestjs/common/interfaces/external/kafka-options.interface";
-import { IBusAdapter } from "../interfaces/bus/bus-adapter.interface";
-import { ITransferData } from "../interfaces/transfer-data";
-import { TransferDataDto } from "../interfaces/transfer-data-dto.interface";
-import { IOnInit } from "../interfaces/lifecycles";
-import { ISetOptions } from "../interfaces/set-options.interface";
-import { loadPackage } from "../utils/load-package.util";
+// import {
```

Comment:
@teogl I had to do it because when I updated nest.js all its interfaces for broker servers were removed :(